libs/luci-lib-jsonc/src/jsonc.luadoc

```diff
@@ -121,10 +121,22 @@ parser:set({ "some", "data" })`
 ]]
 
 ---[[
-Serialize current parser state as JSON.
+Generate an ltn12-compatible sink.
 
 @class function
 @sort 4
+@name parser.sink
+@return Returns a function that can be used as an ltn12 sink.
+@usage `parser = luci.jsonc.new()
+ltn12.pump.all(ltn12.source.file(io.input()), parser:sink())
+print(parser:get())`
+]]
```

Comment:
Do you know what ltn12 is? If not, no wonder you don't understand the documentation. ^^
I first had a `@see http://lua-users.org/wiki/FiltersSourcesAndSinks` line in there, but the LuaDoc doesn't cope well with URLs and tried to interpret it as a method name. So if you tell me how to integrate that link in a reasonable way, I'll happily include it.